include/networkit/graph/Graph.hpp

```diff
@@ -1535,6 +1556,20 @@ class Graph final {
         return outEdges[u][i];
     }
 
+    /**
+     * Return the weight to the i-th (outgoing) neighbor of @a u.
+     *
+     * @param u Node.
+     * @param i index; should be in [0, degreeOut(u))
+     * @return @a edge weight to the i-th (outgoing) neighbor of @a u, or @c +inf if no such
+     * neighbor exists.
+     */
+    edgeweight getIthNeighborWeight(node u, index i) const {
```

Comment:
Is it intentional that this method is not added to the Python interface?

Reply:
`getIthNeighbor` does not have a Python interface, so we either provide a Python interface for both or for none.